### PR TITLE
Added Test data for Catalan and Basque

### PR DESCRIPTION
**Pull request by iolalla**
As propossed by @ojwb here goes the Test Data set for Catalan and Basque, so the CI system doesn't fail and we can have a common testing ground for future improvements. 

Thanks

### CODE REVIEW

File: basque/COPYING [hunk 1]
```diff
@@ -0,0 +1,16 @@
+voc.txt is a mixture of data licensed as BSD 3-clause (as described in
+../COPYING) and CC BY-SA.
+
+voc.txt was derived in part from a dump of the Euskera(Basque) Wikipedia by the
```
Comment (ojwb) [inline basque/COPYING]: Is the above accurate, or is the `basque/voc.txt` file here actually only from Wikipedia data?

The parts about "a mixture of data licensed as BSD 3-clause" and "derived in part" are present for some languages where we've combined another word list with the wikipedia data (e.g. for French we needed a larger list to provide test coverage for a fix so we merged wikipedia data into the existing list).  If the data is only from wikipedia, we should just state that - for example, see `lithuanian/COPYING`.

File: catalan/COPYING [hunk 2]
```diff
@@ -0,0 +1,16 @@
+voc.txt is a mixture of data licensed as BSD 3-clause (as described in
+../COPYING) and CC BY-SA.
+
+voc.txt was derived in part from a dump of the Catalan Wikipedia by the
```
Comment (ojwb) [inline catalan/COPYING]: Same question applies here...

**Comment by iolalla** [comment]
With the last commit I think it reflects the correct copyright notices.

**Comment by ojwb** [comment]
Just to clarify, is each new `voc.txt` only from wikipedia data?

I think that must be the case, but you've left the "in part" phrasing in there which suggests there is also another data source.  I'm happy to fix up the wording, but I want to make sure I'm not changing it incorrectly.

**Comment by iolalla** [comment]
Yes, only from Wikipedia.

I started another list, but  definely worse than Wikipedia, is true that in Wikipedia's list there are words in English, Spanish and French, but it is far more complete than any other list I've tried.

**Comment by ojwb** [comment]
Thanks for clarifying - will get this merged soon.

Generating wordlists from Wikipedia data certainly isn't perfect, but it does seem to give a fairly reasonable quality list with a suitable licence for not much effort.  And arguably it's useful to include some foreign words that might appear in documents of a particular language.

**Pull request merged**